apps/design-system/src/pages/view-preview/view-settings.tsx

```diff
@@ -43,8 +43,9 @@ const ViewSettings: FC<ViewSettingsProps> = ({ routes }) => {
   return (
     <aside className={clsx(css.viewSettings, 'shadow-4')} data-show={showSettings ? 'show' : 'hide'}>
       <Button
-        variant="link_accent"
-        size="icon"
+        variant="link"
```

Comment:
Radix uses a separate component for `Link`. 
Ideally a "Link" should not have an `onClick` on it. It should instead take a `to` prop for navigation. Should this also be a separate component like `LinkButton`?
https://www.radix-ui.com/themes/docs/components/link

Reply:
There are some use cases where we need Button component to behave and look like Link component. Hence added a `link` variant to Button.

<img width="525" alt="image" src="https://github.com/user-attachments/assets/cce6fdb9-5f08-44e8-8e02-94424914f5fd" />
<img width="591" alt="image" src="https://github.com/user-attachments/assets/df2de269-247f-4a2a-93c0-0166077595a7" />

<img width="369" alt="image" src="https://github.com/user-attachments/assets/9a8821f4-061c-476f-9e9a-ce2f93b30302" />